program/tests/duplicate_block_proof.rs

```diff
@@ -571,10 +656,12 @@ async fn missing_sigverify() {
     );
     let err = context
         .banks_client
-        .process_transaction(transaction)
+        .simulate_transaction(transaction)
         .await
-        .unwrap_err()
-        .unwrap();
+        .unwrap()
+        .result
+        .unwrap()
+        .unwrap_err();
```

Comment:
Was this change intended?

Reply:
Not intended, this is leftover from when I was testing out simulate transaction for the hack, removed it and the ones below